### ISSUE TITLE
Feature Request: "Sleeping" ECU data retrieval

### ISSUE THREAD

**Issue by alexsahka** [meatpiHQ/wican-fw#159]
Here's an idea for a vehicle ECU designed to enter sleep mode immediately after the ignition is turned off.

The adapter could save vehicle parameters every minute (or more frequently) when the ignition is on and not connected to MQTT to internal adapter storage.
Upon connecting to home Wi-Fi, when the ignition is in the "off" state and disconnects from the ECU, it would then start publishing the latest saved data to MQTT.

Thank you.

**Comment by njh** [comment]
Yes, please! I was just thinking that this would be a good idea.
When the ignition is on, I am probably not at home.
But when I get home and the ignition is off, it would be good to send the last known data.
